sdks/python/apache_beam/runners/worker/statesampler_fast.pyx

```diff
@@ -109,25 +80,18 @@ cdef class StateSampler(object):
   cdef pythread.PyThread_type_lock lock
 
   cdef public int64_t state_transition_count
-  cdef int64_t time_since_transition
+  cdef int64_t _time_since_transition
```

Comment:
<!--thread_id:cc_162189862_t; commit:4c27aa0fbbbc31e24e3b8d6bc04128b44d18b9ec; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> Why this change? Or should we be changing all the others as well for consistency?

<!--section:body-->
I had thought this was only c-accessible. I've kept it consistent.